core/templates/pages/error-pages/error-page.module.ts

```diff
@@ -13,7 +13,9 @@
 // limitations under the License.
 
 /**
- * @fileoverview Module for the error page.
+ * @fileoverview Module for the error page. This module has not
+ * been migrated to angular router. Remove in favor
```

Comment:
Please write the directive as a TODO and file an issue for it.

Also, just to check, will the error page work in general (since pages besides the admin page use it)? I'm a bit confused because the docstring here says this page hasn't been migrated to Angular, but clearly some significant changes have been done to the error page files.

Reply:
I created two error modules. One to handle existing pages (error codes from the server) and one to handle the new error routes (`oppia.org/error/401`) that are registered wth angular router.

ErrorPageComponent and ErrorRootComponent cannot be declared by two modules so I also created a shared module that declares these two and is used by both of our error modules.

Other changes:
  * Since the error root component is now a child class of the base root component, the error page doesn't need to update the error page title and change the language of the page so that code and such tests have been removed.